README.md

```diff
@@ -950,6 +979,30 @@ again in a future request, it will use it directly from Nexus rather than downlo
 uploading it again. This guarantees that any dependency used for a Cachito request can be used again
 in a future Cachito request.
 
+### RubyGems
```

Comment:
Could you also mention some of the Cachito-specific requirements and how users need to update their repository to meet those requirements?

Off the top of my head, that is
* Gem must pure ruby
* Git deps must use https:// and specify the exact commit hash